projects/centos-7/Dockerfile

```diff
@@ -1,29 +1,31 @@
 FROM centos:7
 
 RUN yum update -y && \
-    yum install -y \
-        which \
-        epel-release \
-        wget \
-        python \
-        python-psutil \
-        python-requests \
-        python-jsonschema \
-        python-pip \
-        rsyslog \
-        gnupg \
-        unixODBC-devel \
-        authd
+  yum install -y \
+    which \
+    epel-release \
+    wget \
+    gcc-c++ \
+    make \
+    python3 \
+    python3-pip \
+    rsyslog \
+    gnupg \
+    unixODBC-devel \
+    authd
+  #&& \
+  #yum clean all && \
+  #rm -rf /var/cache/yum /tmp/* && \
```

Comment:
dead?

Reply:
No, this is related to #25. For some reason, I can't get anything to stick from pip unless I perform the following two steps separately. I could, of course, leave these lines here and then run `yum update` in the following steps, but I think we should just leave this for now. If you disagree, I'm fine to try to fix this now.

Reply:
i was just reflecting on the commented-out code...

Reply:
Right, I'm saying that the commented-out code is clearing the yum cache just to be updated again in the step on line 20 because I need it again to fix the aforementioned issue with pip.

Reply:
okie doke.

Reply:
I will mark this resolved as it will be part of the investigation in the aforementioned linked issue.